pkg/resources/actions.go

```diff
@@ -80,6 +80,25 @@ func Update(w *wildflyv1alpha1.WildFlyServer, client client.Client, objectDefini
 	return nil
 }
 
+func Patch(w *wildflyv1alpha1.WildFlyServer, client client.Client, objectDefinition runtime.Object, patch client.Patch) error {
+	logger := logWithValues(w, objectDefinition)
+	logger.Info("Patching Resource")
+
+	meta := objectDefinition.(metav1.Object)
+	// mark the object with the current wildflyserver generation unless it is the wildflyserver itself
+	if objectDefinition != w {
+		MarkServerGeneration(w, meta)
+	}
+
+	if err := client.Patch(context.TODO(), objectDefinition, patch); err != nil {
+		logger.Error(err, "Failed to patch resource")
+		return err
+	}
+
+	logger.Info("Patched resource")
```

Comment:
@yersan should this be logged at `Debug` level?

Reply:
@jmesnil I've changed the message to debug level for the trace that confirms the action taken. There is no need to show it as info, if the operation fails an error is logged, otherwise, we can assume it was executed successfully, no need to log such a confimation